test/integration/global-teardown-errors.spec.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+
+const assert = require('assert');
+const path = require('path');
+const { spawnSync } = require('child_process');
+const fs = require('fs').promises;
+
+describe('Global Teardown Error Handling', function() {
+  this.timeout(5000);
+
+  const setupFile = 'test/fixtures/global-teardown/setup.js';
+  const testFile = 'test/fixtures/global-teardown/test.js';
+
+  before(async function() {
+    await fs.mkdir(path.dirname(setupFile), { recursive: true });
```

Comment:
[Refactor] Err, most fixture files are checked in to the repository. This would be the first instance in the `test/` directory of `fs.mkdir`. -1 from me on this strategy of writing test files dynamically and then removing them after the tests are done. Please switch to having fixtures checked in as regular files.

Reply:
@TG199 this piece of feedback is still open